docs/usage_guide.rst

```diff
@@ -529,3 +555,13 @@ Local system clock is not synchronized
 Some remote servers (e.g. exchanges) require your local system clock synchronized with theirs. In case of not having it synchronized the request will fail and Rotki will either display a specific error message (i.e. 409 status code and a *local system clock is not sync* message) or the generic 500 error one (please, :ref:`report it to us <bug_reporting>`).
 
 Follow your OS official guidelines about how to synchronize the clock with an Internet Time Server and try again.
+
+Sign-up restoring database fails
+==================================
+
+Please, make sure you are using your premium subscription API keys/secret and the same password.
+
+Data with multiple accounts/devices is not synced
+===================================================
+
+Please, make sure all your accounts have the "Allow data sync with Rotki Server" switched on, and that on each log in you take care of managing your local database as wanted (e.g. restoring you local database with the remote one). See the section `Sync data with Rotki Server <sync_data_with_rotki_server_>`_ for more information about how to sync data with multiple accounts/devices.
```

Comment:
> you take care of managing your local database as wanted

I think I know what you are trying to say here but had to read it 3 times to understand. Can you maybe rewrite more ilke?

on each log in you make the appropriate choice when prompted to replace the local database.